jina/orchestrate/deployments/__init__.py

```diff
@@ -305,7 +308,7 @@ def is_sandbox(self) -> bool:
 
         :return: True if this deployment is provided as a sandbox, False otherwise
         """
-        uses = getattr(self.args, 'uses', '')
+        uses = getattr(self.args, 'uses', '') or ''
```

Comment:
does not make sense this change?

Reply:
Yes because uses now can be None, would you accept either of these changes ?
```suggestion
        uses = getattr(self.args, 'uses') or ''
```
or 
```suggestion
        uses = getattr(self.args, 'uses')
        is_sandbox = isinstance(uses, str) uses.startswith('jinahub+sandbox://')
```

Reply:
first one makes more sense in my opinion, but no need to change, I understand now